go.mod

```diff
@@ -85,12 +82,12 @@ require (
 	google.golang.org/protobuf v1.27.1
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
-	helm.sh/helm/v3 v3.7.2
-	k8s.io/api v0.22.6
-	k8s.io/apiextensions-apiserver v0.22.6
-	k8s.io/apimachinery v0.22.6
-	k8s.io/cli-runtime v0.22.6
-	k8s.io/client-go v0.22.6
+	helm.sh/helm/v3 v3.8.0
+	k8s.io/api v0.23.1
+	k8s.io/apiextensions-apiserver v0.23.1
+	k8s.io/apimachinery v0.23.1
+	k8s.io/cli-runtime v0.23.1
+	k8s.io/client-go v0.23.1
```

Comment:
The command `go mod tidy` insists on updating these deps to this version (which is the one Helm uses). However, it does not really affect us since we are overriding the actual version in the `replace` section.
Given our code is ready to use post-k8s-1.21 deprecations, I'd bet it's safe to update to 1.23 as soon as it becomes available in GKE.